extensions/wikia/EmbeddableDiscussions/EmbeddableDiscussionsController.class.php

```diff
@@ -28,9 +28,67 @@ public static function onBeforePageDisplay() {
 		return true;
 	}
 
+	/**
+	 * Checks arguments for errors.
+	 * @param array $args
+	 * @oaram errorMessage Return parameter with the proper error message to show. Disregard if return is false
+	 * @return true if ok, false if error
+	 */
+	private static function checkArguments( array $args, &$errorMessage ) {
+		// mostrecent must be bool
+		if ( array_key_exists( 'mostrecent', $args ) && $args['mostrecent'] !== 'true' && $args['mostrecent'] !== 'false' ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "mostrecent" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)